utils/config.ts

```diff
@@ -9,25 +9,40 @@ import * as t from "io-ts";
 import { readableReport } from "italia-ts-commons/lib/reporters";
 import { NonEmptyString } from "italia-ts-commons/lib/strings";
 
+export const RedisParams = t.intersection([
+  t.interface({
+    REDIS_URL: NonEmptyString
+  }),
+  t.partial({
+    REDIS_CLUSTER_ENABLED: t.boolean,
```

Comment:
If we have this tyings, we need to remap the value in [here](https://github.com/pagopa/io-functions-cgn/pull/30/files#diff-8f4625a1cf4cb245a2f9e780a9a17ff6a7132b8e070997144a67499d7ff793b4R48)
like
```typescript
const errorOrConfig: t.Validation<IConfig> = IConfig.decode({
  ...process.env,
  isProduction: process.env.NODE_ENV === "production",
  REDIS_CLUSTER_ENABLED: fromNullable(process.env.REDIS_CLUSTER_ENABLED)
    .map(_ => _.toLowerCase() === "true")
    .toUndefined()
});
```